aws/lambda/clickhouse-replicator-dynamo/test_lambda_function.py

```diff
@@ -0,0 +1,11 @@
+import json
+from pathlib import Path
+from lambda_function import handle_event
+
+if __name__ == "__main__":
+    # Uses the sample.json file to test the lambda function.  Does not perform
+    # the insert.  Does not go through error path.  Prints the query so you can
+    # try it out in the console.  Change the database to fortesting to check.
+    with open(Path(__file__).parent / "sample_workflow_job.json") as f:
+        event = json.load(f)
+    handle_event(event, dry_run=True)
```

Comment:
I think we could dump the query string here in dry-run mode for testing, i.e. writing it to `query.sql` for example and check against the expected value in another file, i.e. `sample_query.sql`

Reply:
Done, but its a bit jank